### DISCUSSION TITLE
Zwift - Expected Behaviour

### DISCUSSION THREAD

**Discussion by rjlee** [cagnulein/qdomyos-zwift#2465] [General]
Hi,

My setup is:

qz app running on Samsung Android Phone - v 2.16.55
iFit app running on Lenovo M11 Tab Tablet
NordicTrack S22i running companion app

I've got zwift working and integrated (using the virtual device presented by qz).  Auto-incline seems to work well, however there are no changes to resistance.  Is this the intended behaviour i.e. only incline is automatically adjusted ?

Thanks.

**Comment by cagnulein** [comment]
Yes, exactly it's the intended behavior: incline also affect resistance so
with one action qz handles both
let me know if you have any other questions


Il giorno sab 20 lug 2024 alle 21:00 Robert Lee ***@***.***>
ha scritto:
